docs/docsite/rst/user_guide/playbooks_intro.rst

```diff
@@ -33,34 +33,38 @@ A playbook runs in order from top to bottom. Within each play, tasks also run in
 * the managed nodes to target, using a :ref:`pattern <intro_patterns>`
 * at least one task to execute
 
+.. note::
+
+	In Ansible 2.10 and later, we recommend you use the fully-qualified collection name in your playbooks to ensure the correct module is selected, as some modules may have the same name in different collections. See :ref:`collections_using_playbook`.
```

Comment:
```suggestion
	In Ansible 2.10 and later, we recommend you use the fully-qualified collection name in your playbooks to ensure the correct module is selected, because multiple collections can contain modules with the same name (for example, ``user``). See :ref:`collections_using_playbook`.
```